scripts/migrateHistoricBids.ts

```diff
@@ -115,7 +115,7 @@ function mapBidtoBidPlacedMessage(bid: Bid): TypedMessage<BidPlacedV1Data> {
 }
 
 function mapBidtoBidCancelledMessage(
-  bid: Bid
+  bid: CanceledBid
```

Comment:
Shouldn't this take a `Bid` and not a `CanceledBid`?
The function name is "Bid" to "BidCancelled" message

Reply:
More related to my above question, if this works with either `Bid` or `CanceledBid` then we should take a specific type for this.

Reply:
This is due to the new changes from feedback on the pull request, that CanceledBid should be an interface that extends bid.

This means that Bid doesnt have the canceldate field anymore, and bids have to be cast as canceledBids for them to inherit it.

Reply:
I agree that it should be an interface that extends Bid, but you are passing `Bid`'s into this function, not `CanceledBid`'s

Reply:
very minor nit is that the word `Canceled` seems to have only one `l` in some places but two in others. Googling says both are fine, so pick one and be consistent